src/hub.js

```diff
@@ -111,6 +111,12 @@ import { generateDefaultProfile, generateRandomName } from "./utils/identity.js"
 import { getAvailableVREntryTypes, VR_DEVICE_AVAILABILITY } from "./utils/vr-caps-detect.js";
 import ConcurrentLoadDetector from "./utils/concurrent-load-detector.js";
 
+window.RENDER_ORDER = {
```

Comment:
can this be scoped under the [`window.APP` global](https://github.com/mozilla/hubs/blob/1b3c30b1/src/hub.js#L78)?

Reply:
yes